### ISSUE TITLE
Support Rust source code 🦀

### ISSUE THREAD

**Issue by command-tab** [sbarex/SourceCodeSyntaxHighlight#31]
Syntax Highlight appears to support Rust source code files, but the extension it's looking for is incorrect. It has them as `.rust` files, but Rust uses `.rs`. Here's the `mdls` output:

```
$ mdls -name kMDItemContentType -name kMDItemContentTypeTree main.rs 
kMDItemContentType     = "org.rust-lang.rust"
kMDItemContentTypeTree = (
    "org.rust-lang.rust",
    "public.source-code",
    "public.plain-text",
    "public.text",
    "public.data",
    "public.item",
    "public.content"
)
```

Thank you for this app! It's wonderful to have syntax-highlighted QuickLook support once again under Catalina ❤️

**Comment by sbarex** [comment]
Okay, thanks for the report. In the next version (hopefully released soon) there will be the correction.

**Comment by sbarex** [comment]
Fixed on last release (1.0.b20)

**Comment by command-tab** [comment]
It works! Thank you! 🎉
